File: Calculadora.py
#Calculadora simples com as 4 operações

numero1 = 0
numero2 = 0
resultado = 0
operacao = ''

numero1 = int(input('Digite o número 1: '))
operacao = input('Digite a operação: ')
numero2 = int(input('Digite o número 2: '))

while True:
    try:
        if operacao == '+':
            resultado = numero1 + numero2
        elif operacao == '-':
            resultado = numero1 - numero2
        elif operacao == '/':
            resultado = numero1 / numero2
        elif operacao == '*':
            resultado = numero1 * numero2
        else:
            resultado = 'Operação inválida!'
    except:
        print('Tente novamente :)')
    print(str(numero1) + ' ' + str(operacao) + ' ' + str(numero2) + '=' + str(resultado))
    break

File: Calculadora.rb
#Calculadora simples com as 4 operações

puts "Digite o número 1:"
numero1 = gets.chomp.to_f

puts "Digite o número 2:"
numero2 = gets.chomp.to_f

puts "Escolha a operação: "
puts "1 - Adição"
puts "2 - Subtração"
puts "3 - Multiplicação"
puts "4 - Divisão"

operacao = gets.chomp.to_i

if operacao == 1
  resultado = numero1 + numero2
elsif operacao == 2
  resultado = numero1 - numero2
elsif operacao == 3
  resultado = numero1 * numero2
elsif operacao == 4
  resultado = numero1 / numero2
else
  puts "Operação inválida."
end

puts "O resultado da operação é: #{resultado}"

#TESTE NA NOVA BRANCH
#TESTE NA MESMA BRANCH
